### PR TITLE
 pull request for n60a pcb

### PR DESCRIPTION
**Pull request by valdiieee**
## Description

i wanna merge firmware for n60a pcb by nix studio
## Types of Changes
new firmware

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Does not compile:
* `default` keymap missing
* `lite` is no longer a valid bootmagic option

File: keyboards/nix_studio/n60a/keymaps/via/n60a_via.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
+{
```
Comment (drashna) [inline keyboards/nix_studio/n60a/keymaps/via/n60a_via.json:1]: This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards)

Also, you may have noticed that "via.json" files are not included by default, and you probably had to rename the file to get it to be added.  This is intentional, for exactly this reason.

File: keyboards/nix_studio/n60a/n60a.c [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "n60a.h"
```
Comment (drashna) [inline keyboards/nix_studio/n60a/n60a.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/nix_studio/n60a/n60a.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Nix Studio
```
Comment (drashna) [inline keyboards/nix_studio/n60a/n60a.h:1]: License type needs to be specified. 
```suggestion
// Copyright 2022 Nix Studio
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/nix_studio/n60a/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# N60_A
+
+60% PCB for Sofia keyboard
+
+* Keyboard Maintainer: [Nix Studio](https://github.com/Nix-Studio) & [Valdiieee](https://github.com/valdiieee)
+* Hardware Supported: N60_A
+* Hardware Availability: Groupbuy
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/nix_studio/n60a/readme.md:9]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/nix_studio/n60a/rules.mk [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/nix_studio/n60a/rules.mk:18]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/nix_studio/n60a/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______  
+    ),
+    [2] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______  
+    ),
```
Comment (fauxpark) [inline keyboards/nix_studio/n60a/keymaps/via/keymap.c:41]: Needs one more layer, or `#define DYNAMIC_KEYMAP_LAYER_COUNT 3` in `config.h`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/nix_studio/n60a/keymaps/via/keymap.c (hunk 6 above)
Comment (valdiieee) [reply keyboards/nix_studio/n60a/keymaps/via/keymap.c:41]: done

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**